netsync/manager.go

```diff
@@ -733,6 +739,13 @@ func (sm *SyncManager) handleInvMsg(imsg *invMsg) {
 	if err != nil {
 		log.Errorf("Failed to send invs from queue: %s", err)
 	}
+
+	if haveUnknownInvBlock && !sm.current() {
+		// If one of the  an inv message is an unknown block
```

Comment:
Double space between `the  an`. Fix quickly before Stas get's post-trauma from his war with Mr. Collins